docs/en/latest/deployments/ack.md

```diff
@@ -38,6 +38,8 @@ This document explains how to install Ingress APISIX on [ali-cloud ACK](https://
 ```shell
 cd /path/to/apisix-helm-chart
 helm repo add bitnami https://charts.bitnami.com/bitnami
+helm repo add apisix https://charts.apiseven.com
+# Use `hem search repo apisix` to search charts about apisix
 helm dependency update ./charts/apisix
 helm install apisix ./charts/apisix \
```

Comment:
This line should be change also.

```shell
helm install apisix apisix/apisix
```

That's why we should add a comment `# Use `hem search repo apisix` to search charts about apisix`, because the charts name `apisix/apisix` can be found from this command.

Reply:
Like tihs?

```shell
cd /path/to/apisix-helm-chart
helm repo add bitnami https://charts.bitnami.com/bitnami
helm repo add apisix https://charts.apiseven.com
# Use `hem search repo apisix` to search charts about apisix
helm dependency update ./charts/apisix
helm install apisix apisix/apisix \
  --set gateway.type=LoadBalancer \
  --set allow.ipList="{0.0.0.0/0}" \
  --namespace ingress-apisix \
kubectl get service --namespace ingress-apisix
```

Reply:
Try this :), and we suggest you to run it locally
```suggestion
cd /path/to/apisix-helm-chart
helm repo add bitnami https://charts.bitnami.com/bitnami
helm repo add apisix https://charts.apiseven.com
# Use `hem search repo apisix` to search charts about apisix
helm repo update
helm install apisix apisix/apisix \
  --set gateway.type=LoadBalancer \
  --set allow.ipList="{0.0.0.0/0}" \
  --namespace ingress-apisix \
kubectl get service --namespace ingress-apisix
```